beast/fitting/pdf2d.py

```diff
@@ -19,29 +19,26 @@ def __init__(
         maxval_p2=None,
     ):
         """
-        Create an object which can be used to efficiently generate a 1D pdf for an observed object
+        Create an object which can be used to efficiently generate a 2D pdf
+        for an observed object
 
         Parameters
         ----------
-
-        gridvals_p1, gridvals_p2: array-like
-            values of the quantity for all the grid points
-
-        nbins_p1, nbins_p2: int
-            number of bins to use for each dimension of the 2D PDF
-
-        logspacing_p1, logspacing_p2: bool
-            whether to use logarithmic spacing for the bins of each dimension
-
-        minval_p1, maxval_p1, minval_p2, maxval_p2: float (optional)
+        gridvals_p1, gridvals_p2 : ndarray
+            1D `float` array with the values of the quantity for all the grid points
+        nbins_p1, nbins_p2 : int
+            number of bins to use for the 1D pdf
```

Comment:
not 2D?

Reply:
Nope.  The model SED grid is just a list of models and parameters.  Hence, when the grid values are passed, they are 1D arrays.  They are in order, so the correlations are preserved.